task6.rb

```diff
@@ -1,8 +1,18 @@
+class NotANumberError < StandardError
```

Comment:
Please check what [task](https://github.com/MastersAcademy/ruby-course-2019/blob/master/ruby_basic.md#task-6-) asks you to do.